camayoc/tests/qcs/cli/utils.py

```diff
@@ -155,6 +156,48 @@ def source_show(options, output, exitstatus=0):
     assert qpc_source_show.exitstatus == exitstatus
 
 
+def scan_add(options):
+    """Run ``qpc scan add`` command with ``options`` returning its output.
```

Comment:
Docstring does not match what this function is doing. It returns None and also can raise AssertionError